src/components/Check/Check.scss

```diff
@@ -49,19 +61,34 @@ $checkBoxHeight: 20px;
 }
 
 .ms-Check-background {
+  font-family: 'FabricMDL2Icons';
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
   &:before {
+    content: '\E91F';
+    color: #ffffff;
+  }
+
+  &:after {
+    content: '\EA3A';
+    color: $ms-color-neutralTertiaryAlt;
+  }
+
+  &:before,
+  &:after {
+    @include left(-2px);
```

Comment:
Offset is there to center the checkmark in the middle of the circle.  See screenshot of below with left: 0px

![screen shot 2016-09-28 at 10 42 22 am](https://cloud.githubusercontent.com/assets/1291968/18925305/5582a17e-8568-11e6-982c-6eaef749c0e7.png)